docs/content/configuration/configuring_hostdefiner.md

```diff
@@ -0,0 +1,12 @@
+# Configuring the host definer
+
+Some of the parameters within the HostDefiner custom resource are configurable. Use this information to help decide whether the parameters for your storage system need to be updated.
+
+For more information about using the host definer, see [Using dynamic host definition](../using/using_hostdefinition.md).
+    
+|Field|Description|
+|---------|--------|
+|`prefix`|Adds a prefix to the hosts defined by the host definer.|
```

Comment:
We should add a note of the max chars and chars types supported for host name in the storage (currently only for SVC).

Reply:
@roysahar  @matancarmeli7  what are the max characters and allowed character types here?
regarding SVC limitation - this is documented in limitations

Reply:
I will check and update this

Reply:
this is Carlos's answer:
"63 characters, must start alpha, characterset is printable characters (not sure what special characters are allowed)"

Reply:
@erantzabari  how do we want to define this in the guide (if at all)
@oriyarde 
note that in https://www.ibm.com/docs/en/flashsystem-9x00/8.5.x?topic=commands-host host_name only states that it needs to be alphanumeric. is this the relevant place to look?

Reply:
@oriyarde I don't know, I am not sure.
So if we allow the user to put 20 chars as a prefix, he won't be able to have a host with more than 42 chars in his name?

Reply:
if we don't have a clever "final name" logic, and the storage command fails for more than 63 chats, then it seems like yes

Reply:
could we write 20 chars for now, until we have a final decision? we cannot afford to miss this

Reply:
We can add a comment that prefix and host_name are bounded to the limitation of the storage system and the rules from the storage system applies also for the prefix.

Reply:
updated per eran's input. @oriyarde please approve
https://github.com/IBM/ibm-block-csi-driver/blob/83cbf8101de7752cd6afa57f8ee1fbd70bdd1278/docs/content/configuration/configuring_hostdefiner.md

Reply:
fine, but this means that if the user wants a combination of prefix, separator and worker name which is longer than want the storage allows, we could never add a clever hashing mechanism (as we did for volumes and snapshots)

Reply:
@rivkap , we can add comment in the documentation on top of the comment that we bound to the limitation of the storage, that the prefix is part of the host name (so the length is combined to both the prefix and the name given).

Reply:
@oriyarde , I wouldn't go to hashing solution. We added it to DS8K due to very short volume name - 63 chars are good enough for prefix and name.

Reply:
thanks. updated per @erantzabari 's input

Reply:
@erantzabari
1. we added volume name hashing to all storage systems
2. a valid k8s hostname can be up to 253 chars long, even without a prefix
3. in the future, we might add host definer support for stricter storage systems

we can always remove the prefix length limitation, but we cannot add retroactively. I don't see why not being on the safe side...

Reply:
We added the limitation of the length according to the storage system (not specific number).
Do you suggest something else or you good with that?

Reply:
I suggest something else: [limit the prefix length](https://github.com/IBM/ibm-block-csi-driver/pull/517#discussion_r921226599) (as we did with volume prefix), to allow adding hashing in the future, otherwise a long prefix would render the hashing useless (due to collisions)